File: app/src/main/java/com/lenecoproekt/minesweeper/GameObject.java
package com.lenecoproekt.minesweeper;

public class GameObject {
    public int x, y, countMineNeighbors;
    public boolean isMine, isOpen, isFlag;
    public GameObject (int x, int y, boolean isMine) {
        this.x = x;
        this.y = y;
        this.isMine = isMine;
    }
}

File: app/src/main/java/com/lenecoproekt/minesweeper/logic/GameObject.kt
package com.lenecoproekt.minesweeper.logic

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
class GameObject(var x: Int, var y: Int, var isMine: Boolean) : Parcelable {
    var countMineNeighbors = 0
    var isOpen = false
    var isFlag = false
}